Dockerfile

```diff
@@ -3,22 +3,37 @@ FROM $DOCKER_IMAGE AS builder
 
 ENV MINETEST_GAME_VERSION master
 ENV IRRLICHT_VERSION master
+ENV SPATIALINDEX_VERSION 1.9.3
+ENV LUAJIT_VERSION v2.1
 
-RUN apk add --no-cache git build-base cmake sqlite-dev curl-dev zlib-dev zstd-dev \
-		gmp-dev jsoncpp-dev postgresql-dev ninja luajit-dev ca-certificates
+RUN apk add --no-cache git build-base cmake curl-dev zlib-dev zstd-dev \
+		sqlite-dev postgresql-dev hiredis-dev leveldb-dev \
+		gmp-dev jsoncpp-dev ninja ca-certificates
 
 WORKDIR /usr/src/
-RUN git clone --recursive https://github.com/jupp0r/prometheus-cpp/ && \
+RUN cd /usr/src/ && \
+	git clone --recursive https://github.com/jupp0r/prometheus-cpp/ && \
 	cd prometheus-cpp && \
 	cmake -B build \
 		-DCMAKE_INSTALL_PREFIX=/usr/local \
 		-DCMAKE_BUILD_TYPE=Release \
 		-DENABLE_TESTING=0 \
 		-GNinja && \
 	cmake --build build && \
-	cmake --install build
-
-RUN git clone --depth=1 https://github.com/minetest/irrlicht/ -b ${IRRLICHT_VERSION} && \
+	cmake --install build && \
+	cd /usr/src/ && \
```

Comment:
Would it be possible to split individual configuration steps (prometheus, spatialindex, luajit, irrlicht) ? That would improve the readability a lot.

Reply:
Did you mean, split them into separate `RUN` sections? That would increase the amount of layers that are generated for the image. So I would be against that.

But I can additional new lines and indentions for readability. Would that satisfy you?

Reply:
[4209576](https://github.com/minetest/minetest/pull/12914/commits/4209576b8d5d2a2fa5070b17d8167efbf6801ce7)
Thoughts?

Reply:
Thank you. That's a sufficient improvement.